pandas/tests/series/test_sorting.py

```diff
@@ -64,6 +64,24 @@ def test_sort_values(self):
         ordered = ts.sort_values(ascending=False, na_position='first')
         assert_almost_equal(expected, ordered.valid().values)
 
+        # ascending=[False] should behave the same as ascending=False
+        ordered = ts.sort_values(ascending=[False])
+        expected = np.sort(ts.valid().values)[::-1]
```

Comment:
Could you elaborate? I don't understand what you mean by constructing literally. Do you want me to create an example series for expected and then compare with ordered instead of creating it via np.sort? If you want to avoid NumPy, wouldn't it be better to simply compare the results of ascending=False and ascending=[False] to be _series_equal?